trace/cloud-trace-demo-app/app/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.8.0
+
+RUN pip3 install --upgrade pip
+
+RUN apt-get update \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+COPY requirements.txt /app/
+
+WORKDIR /app
+RUN pip3 install -r requirements.txt
+RUN pip3 install opencensus-ext-stackdriver
+RUN pip3 install grpcio
+COPY *.py /app/
```

Comment:
nit: It's common to just copy the entire folder, (e.g. `COPY . ./`) in cause there are non `.py` files needed for the build.

Reply:
Done.